websites/I/Incels.is/presence.ts

```diff
@@ -0,0 +1,129 @@
+const presence = new Presence({
+		clientId: "1094962047906758796",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Logo = "https://incels.is/logo/512x512.png",
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			startTimestamp: browsingTimestamp,
+		},
+		threadTitleElement = document.querySelectorAll(
+			".p-title-value"
+		)[0] as HTMLHeadingElement,
```

Comment:
Use generics rather than `as` typing. Also, for selectors like this, `querySelector` can be used instead of `querySelectorAll`

Reply:
Done